docs/docs/recipes/manage-users/on-admin-console-page.md

```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
 
 ## Add user
 
-TODO: admin manually add
+Not only can the end-user sign up by his/herself, but also the admin users can manually add accounts for them.
```

Comment:
Sth like, `Admin users could create a new account through Admin Console in the name of an end-user. ` or simply `Admin users could create an account for a new user upon their request through Admin Console.`

Reply:
him/herself BTW...

Reply:
Updated. PTAL.